enhancements/olm/operator-bundle.md

```diff
@@ -105,17 +105,30 @@ An operator author can also specify the version of the format used inside the bu
 operators.operatorframework.io.bundle.mediatype: "registry+v1"
 ```
 
+###### Graph Metadata
+An additional point of concern is the fact that the current [operator-registry manifest format](https://github.com/operator-framework/operator-registry#manifest-format) also has a multi-version aggregation file `package.yaml` that describes data about the channels that individual versions are subscribed to. Given that we are splitting up these manifests into separate objects, we now need a way to define how an individual version subscribes to a given channel. To do this, we will create a new metadata file that defines the channels explicitly for each version called `channels.yaml`
```

Comment:
This file also contained the `packageName` what is the unique identifier to continue to tie a bundle to an old version of that bundle?

Reply:
@shawn-hurley This is a good point. For now, I redefined this to be `registry.yaml` and defined it as a denormalization of the `package.yaml` file to just define data for a specific version of the operator. Please take another look.